Remotion/Web/Development.WebTesting/CoypuElementScopeFillInWithAndSendKeysExtensions.cs

```diff
@@ -105,11 +105,24 @@ private static void SetValueUsingSendKeys ([NotNull] this ElementScope scope, [N
       value = value.Replace("\r", "");
 
       var clearTextBoxWithoutTriggeringPostBack = Keys.Control + "a" + Keys.Control + Keys.Delete;
-      value = clearTextBoxWithoutTriggeringPostBack + value;
 
-      s_log.DebugFormat("FillInWith without triggering PostBack on clear: '{0}'.", value);
+      //GeckoDriver cannot handle the delete command and inputting the value in the same SendKeys after Firefox v99 (would lead to empty field).
+      if (scope.Browser == Browser.Firefox)
```

Comment:
Please use BrowserConfigurationExtensions instead
```suggestion
      if (scope.Browser.IsFirefox())
```
That way we avoid logic duplication.
(Disclaimer: unless there's technical reason against this?)